vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridElement.java

```diff
@@ -430,4 +430,68 @@ public void clickSelectAll() {
                 .id("selectAllCheckbox");
         selectAllCheckbox.click();
     }
+
+    /**
+     * Gets the rows specified by the lower and upper row indexes.
+     *
+     * @param firstRowIndex
+     *            the lower row index to be retrieved (inclusive)
+     * @param lastRowIndex
+     *            the upper row index to be retrieved (inclusive)
+     * @return a {@link GridTRElement} list with the rows contained between the
+     *         given coordinates.
+     */
+    public List<GridTRElement> getRows(int firstRowIndex, int lastRowIndex) {
```

Comment:
The following test doesn't pass:
```java
// From firstRowIndex 0 to lastRowIndex 4 = (0, 1, 2, 3, 4)
var rows = grid.getRows(0, 4);
Assert.assertEquals(5, rows.size());
```

because even though the documentation claims so, `lastRowIndex` isn't inclusive. Could update the implementation to: 
```java
return IntStream.rangeClosed(firstRowIndex, lastRowIndex)
        .mapToObj(this::getRow).collect(Collectors.toList());
```